src/frontend/src/flows/recovery/displaySeedPhrase.ts

```diff
@@ -14,30 +14,18 @@ const displaySeedPhraseTemplate = ({
   copyPhrase: () => Promise<void>;
   onContinue: () => void;
 }) => {
-  const phraseElement: Ref<HTMLElement> = createRef();
   const phraseCopyElement: Ref<HTMLElement> = createRef();
 
   const continueButton: Ref<HTMLButtonElement> = createRef();
   const checkbox: Ref<HTMLInputElement> = createRef();
 
-  // Selects the phrase content to give visual feedback of what's been copied
-  const selectPhrase = () => {
-    withRef(phraseElement, (phraseElement) => {
-      const selection = window.getSelection();
-      if (selection !== null) {
-        const range = document.createRange();
-        range.selectNodeContents(phraseElement);
-        selection.removeAllRanges();
-        selection.addRange(range);
-      }
-    });
-  };
+  // Assume the phrase is a list of space-separated words
+  const recoveryWords = seedPhrase.split(" ");
```

Comment:
does `" "` finds somewhere else in the code base for the seed phrase and if yes, maybe extract a constant?

should you add a test that the count of words is exactly the expected count (25)?

Reply:
> does " " finds somewhere else in the code base for the seed phrase and if yes, maybe extract a constant?

Not really in any meaningful way, but I'll keep that in mind if that's the case in the future

> should you add a test that the count of words is exactly the expected count (25)?

exact count is not important, also we'd then be testing a third party lib (we don't generate the words ourselves) so I would rather not

Reply:
ok